### PR TITLE
Polyfilling fetch and object-assign

### PR DESCRIPTION
**Pull request by janicklas-ralph**

### CODE REVIEW

**Comment by prateekbh** [comment]
what is preventing babel to add `object-assign.js` polyfill in the rest of the bundles?

**Comment by janicklas-ralph** [comment]
Added an alias for babel added object-assign now. LMK if this works

File: packages/next/build/webpack-config.ts [hunk 1]
```diff
@@ -160,6 +166,20 @@ export default async function getBaseWebpackConfig(
       next: NEXT_PROJECT_ROOT,
       [PAGES_DIR_ALIAS]: pagesDir,
       [DOT_NEXT_ALIAS]: distDir,
+      ...(!isServer
+        ? {
+            unfetch: path.join(__dirname, 'polyfills', 'fetch.js'),
+            'isomorphic-unfetch': path.join(__dirname, 'polyfills', 'fetch.js'),
+            'whatwg-fetch': path.join(__dirname, 'polyfills', 'fetch.js'),
+            'object-assign': path.join(
```
Comment (prateekbh) [inline packages/next/build/webpack-config.ts]: will this get re-used here: https://github.com/azukaru/next.js/pull/142/files#diff-dd19f84f6905f5f026b78648a481f0b5R2?

Comment (janicklas-ralph) [reply packages/next/build/webpack-config.ts]: Yup! Good catch! Fixed this.

**Review by prateekbh — APPROVED** [review]

**Review by prateekbh — CHANGES_REQUESTED** [review]
201 files changed! something seems off

**Pull request closed without merging**